docs/v0.4.0/_src/tutorials/conf.py

```diff
@@ -2,7 +2,7 @@
 #
 # This file only contains a selection of the most common options. For a full
 # list see the documentation:
-# https://www.sphinx-doc.org/en/master/usage/configuration.html
+# https://www.sphinx-doc.org/en/main/usage/configuration.html
```

Comment:
This must not be renamed too. I would suggest that you search through for sphinx-doc.org in the code base to check that all these links contain master instead of main.